### ISSUE TITLE
Generate a Dash Docset

### ISSUE THREAD

**Issue by triangulum** [lsst/pipelines_lsst_io#137]
I generally browse programming documentation offline, which makes accessing and searching it much more comfortable. I use [Dash](https://kapeli.com) on macOS and [Zeal](http://velocity.silverlakesoftware.com/) on Linux. Dash has many docsets for things like Python, numpy and astropy, but something that's missing so far is the LSST user guide.

Would it be possible to generate an official LSST "docset" for Dash?

Given the documentation is Sphinx-based, this should be a streamlined process using the existing [doc2dash](https://pypi.org/project/doc2dash/) framework. The steps are all outlines in the Dash [Docset Generation Guide](https://kapeli.com/docsets).

This would be a great community resource to have!

**Comment by jonathansick** [comment]
Sure, I'll look into this.

**Comment by triangulum** [comment]
Thanks!

As an additional request: it would really be nice to have the [Doxygen](http://doxygen.lsst.codes/stack/doxygen/x_masterDoxyDoc/index.html) as a docset too. It's often much more detailed than the Sphinx documentation, and easier to find and browse classes and methods.

**Comment by jonathansick** [comment]
We're in the process of sunsetting the Doxygen documentation, so we won't be making a Dash docset of it. The good news is that all the content that was formerly available in Doxygen, including C++ documentation, will be available from the Sphinx documentation once that transition is complete.
